src/failures/src/communication/readers/queued-sim-var-reader.test.ts

```diff
@@ -0,0 +1,53 @@
+import { QueuedSimVarReader } from '.';
+import { SimVarReaderWriter } from '..';
+
+test('does not read a value that is not in the collection', async () => {
```

Comment:
In jest it's common to put tests in a file or belonging to the same testable thing in a [describe block](https://jestjs.io/docs/api#describename-fn). Personally I find it makes the console output much more readable.

So assuming these tests are for the QueuedSimVarReader:

```suggestion
describe('QueuedSimVarReader ', () => {
    test('does not read a value that is not in the collection', async () => {
```

Reply:
Funny how I've [done this before](https://github.com/davidwalschots/angular-reactive-validation/blob/master/angular-reactive-validation/src/validation-messages/validation-messages.component.spec.ts) but somehow didn't in this project. 😅